README.md

```diff
@@ -30,7 +30,7 @@ Deploy and log in with username *admin* and the password that's set up during de
 
  * Apache **2.4** or higher
  * MySQL >= 5.7 (or MariaDB >= 10.3) 
- * PHP <b>7.3</b> or higher
+ * PHP <b>7.4</b> or higher
```

Comment:
LORIS still works with 7.3, 7.4 isn't a requirement. This shouldn't be changed (Travis runs both 7.3 and 7.4), but install instructions shouldn't be telling people to install a version that will soon be out of date.

(Same with other sections of this PR that are now modified.)